{"uuid": "c5dba024-3e8a-499a-9385-9514a00bee5b", "children": ["782dbe89-4f6e-434e-b763-19351996f5b8"], "befores": [{"name": "pre_conditions_tc_007", "status": "broken", "statusDetails": {"message": "serial.serialutil.SerialException: could not open port 'COM7': PermissionError(13, 'Access is denied.', None, 5)\n", "trace": "  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\aldo.madrid\\Documents\\AutomationExample\\Scripts\\Demo_Code\\test_004_demo_4.py\", line 138, in pre_conditions_tc_007\n    rly = Relay16CH('COM7')\n  File \"C:\\Users\\aldo.madrid\\Documents\\AutomationExample\\Hardware\\relay_usb.py\", line 6, in __init__\n    self._ser = serial.Serial(port=com_port, baudrate=baud_rate)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\serial\\serialwin32.py\", line 33, in __init__\n    super(Serial, self).__init__(*args, **kwargs)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\serial\\serialutil.py\", line 244, in __init__\n    self.open()\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\serial\\serialwin32.py\", line 64, in open\n    raise SerialException(\"could not open port {!r}: {!r}\".format(self.portstr, ctypes.WinError()))\n"}, "start": 1637711436578, "stop": 1637711437598}], "start": 1637711436578, "stop": 1637711437637}